shared/signup/common.tsx

```diff
@@ -66,9 +74,9 @@ const Header = (props: HeaderProps) => (
     fullWidth={true}
     style={Styles.collapseStyles([styles.headerContainer, props.style])}
   >
-    {props.showInfoIcon && (
+    {(props.showInfoIcon || props.showInfoIconRow) && (
       <Kb.Box2 direction="horizontal" fullWidth={true} style={styles.infoIconContainer}>
-        <InfoIcon invisible={props.negative as boolean} />
+        <InfoIcon invisible={(props.negative as boolean) || (props.showInfoIconRow && !props.showInfoIcon)} />
```

Comment:
`|| !props.showInfoIcon` is enough right?

Reply:
`showInfoIconRow` just takes up the space, otherwise the UI jumps around on transition